<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeRenderer</class>
 <widget class="QWidget" name="ShapeRenderer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="shapeLabel">
     <property name="text">
      <string>Shape</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="outlineThicknessLabel">
     <property name="text">
      <string>Outline Thickness</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="outlineThicknessValue"/>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="outlineStyleLabel">
     <property name="text">
      <string>Outline Style</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="sizeValue"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="outlineColorLabel">
     <property name="text">
      <string>Outline Color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="outlineStylePicker">
     <item>
      <property name="text">
       <string>Solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dash</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dot</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="fillColorLabel">
     <property name="text">
      <string>Fill Color</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="shapePicker">
     <item>
      <property name="text">
       <string>Rectangle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Elipsis</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QToolButton" name="toolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Shape Renderer</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::DownArrow</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
